packages/agents/src/subagents/utils.ts

```diff
@@ -8,22 +8,22 @@ import {
 } from "@evo-ninja/agent-utils";
 import { SubAgentContext } from "./SubAgent";
 
-interface CreateScriptExecutorArgs<TAgentContext> {
+interface ScriptExecutorArgs<TAgentContext> {
   context: TAgentContext
   scriptName: string;
   onSuccess: (params: any, result?: string) => AgentFunctionResult;
   onFailure: (params: any, error: string) => AgentFunctionResult;
 }
 
-export const createScriptExecutor = <TAgentContext extends SubAgentContext = SubAgentContext>(
-  args: CreateScriptExecutorArgs<TAgentContext>
+export const scriptExecutor = <TAgentContext extends SubAgentContext = SubAgentContext>(
+  args: ScriptExecutorArgs<TAgentContext>
 ) => {
   return async (params: any): Promise<Result<AgentFunctionResult, string>> => {
```

Comment:
Technically, the "script executor" is this function here (`async (params: any): Promise<Result<AgentFunctionResult, string>>`). The parent one is the one that creates it

Reply:
sounds good